R/FilterStates.R

```diff
@@ -477,13 +479,6 @@ FilterStates <- R6::R6Class( # nolint
                   teal_slice(dataname = private$dataname, varname = input$var_to_add)
                 )
               )
-              logger::log_debug(
```

Comment:
logs on exit are relevant but not worth it to double number of logs. We log only on entry only so we can keep track of the process order